.eslintrc.json

```diff
@@ -1,16 +1,23 @@
 {
   "parserOptions": {
-    "ecmaVersion": 5
+    "ecmaVersion": 6,
+    "sourceType": "module"
   },
   "extends": "eslint:recommended",
   "env": {
-    "commonjs": true
+    "es6": true
   },
   "rules": {
-    "strict": [2, "global"],
+    "strict": [
+      2,
+      "global"
+    ],
```

Comment:
I'm using `jq` to do the updates in this file, but it causes this unneeded diff. Unless someone has a better idea for how to update the file, I think we just let these changes in.

Reply:
I don't personally care about the changed diff much, but could we also just find/replace `commonjs` -> `es6` if that's the direction we're taking?

Reply:
True, that could work. However, we also have to add `sourceModule: "type"`, which does require knowing a bit more about the file. I could insert that content at a specific line. Let me try that out.

Reply:
Inserting the line works, but it'll go wrong if any of the `.eslintrc.json` files are inconsistent.